tests/cloud/test_infer.py

```diff
@@ -0,0 +1,73 @@
+

```

Comment:
When Adding License, please follow the same structure in all files, keep one line space after the license similar to QEfficient/transformers/models/gptj/modeling_gptj.py, also change the year to only 2024 in all files.

Reply:
Good catch, @abukhoy you can use PL and CL from constants or set to 32/128 pls

Reply:
okay, then i am setting PL to 32 and CL to 128 only as of now.